region/maridia/outer/Mt. Everest.json

```diff
@@ -2805,6 +2813,66 @@
         "Too many pixels: left is a fall, right you clip through the crab, low you miss the jump and fall, high is a bonk or crab clip."
       ]
     },
+    {
+      "link": [10, 4],
+      "name": "Leave With Spark",
+      "requires": [
+        "Gravity",
+        {"or": [
+          "SpaceJump",
+          "canGravityJump",
+          {"and": [
+            "HiJump",
+            {"or": [
+              "canWalljump",
+              "canTrickyDashJump"
+            ]}
+          ]}
+        ]},
+        {"canShineCharge": {
+          "usedTiles": 20,
+          "gentleUpTiles": 3,
+          "gentleDownTiles": 1,
+          "steepUpTiles": 1,
+          "openEnd": 0
+        }},
+        {"shinespark": {"frames": 8}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "note": [
+        "Gain a shinecharge on the upper right ledge.",
+        "Use a gravity jump, Space Jump, or HiJump to approach the door above and spark out."
```

Comment:
Not really sure how hard this is, because I'm failing the charge, but ...
Is it worth noting which direction to run? (It may be obvious after trying the strat)
Needs shinecharge movement (or complex?)
I'm assuming it needs midair shinespark as well?